guppylang/checker/errors/linearity.py

```diff
@@ -160,17 +161,17 @@ class MakeOwned(Help):
 class MoveOutOfSubscriptError(Error):
     title: ClassVar[str] = "Subscript {kind.subjunctive}"
     span_label: ClassVar[str] = (
-        "Cannot {kind.indicative} a subscript of `{parent}` with "
-        "{parent.ty.ownership_kind} type `{parent.ty}`"
+        "Cannot {kind.indicative} a subscript of `{parent}` with non-copyable type "
+        "`{parent.ty}`"
     )
     kind: UseKind
     parent: Place
 
     @dataclass(frozen=True)
     class Explanation(Note):
         message: ClassVar[str] = (
-            "Subscripts on {parent.ty.ownership_kind} types are only allowed to be "
-            "borrowed, not {kind.subjunctive}"
+            "Subscripts on non-copyable types are only allowed to be borrowed not "
```

Comment:
You dropped this
`,`ノ( º _ ºノ)
```suggestion
            "Subscripts on non-copyable types are only allowed to be borrowed, not "
```